starknet_py/net/client_models.py

```diff
@@ -332,7 +334,16 @@ class ContractClass:
 
     program: dict
     entry_points_by_type: EntryPointsByType
-    abi: Optional[AbiDictList] = None
+    abi: Optional[AbiDictList]
```

Comment:
I think technically this is a breaking change, please include it in the migration guide.

Reply:
What if we leave `= None` here and make `abi` in **CompiledContract** default to an empty list?